libraries/rush-lib/assets/rush-init/subspaces.json

```diff
@@ -0,0 +1,30 @@
+/**
+ * This is the main configuration file for Rush.
+ * For full documentation, please see https://rushjs.io
+ */
+ {
+  /**
+   * Temporarily use the default schema until the subspaces schema is uploaded
+   */
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
I think you can just put the real path name here. Rush doesn't process it; it is only used by VS Code IntelliSense underlining

Reply:
do you mean a local link to the schema?

Reply:
No, use this string:
```
  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/subspaces.schema.json",
```

After this PR is merged, we will publish your **subspaces.schema.json** file to that location.

FYI the GitHub source is here:

https://github.com/microsoft/json-schemas/tree/main/rush/v5

Reply:
Changed